app/views/restaurant/show.html.erb

```diff
@@ -1,5 +1,6 @@
     <section id='restaurant'>
       <h1 id='name'><%= @restaurant.name %></h1>
+      <%= link_to "Restaurant Information", @restaurant.url, target: :_blank %>
```

Comment:
Add link for users to see more information than we present about the restaurant.